tests/ui/abi/unsupported.i686.stderr

```diff
@@ -1,222 +1,123 @@
-warning: the calling convention "ptx-kernel" is not supported on this target
-  --> $DIR/unsupported.rs:38:15
+error[E0570]: `"ptx-kernel"` is not a supported ABI for the current target
```

Comment:
Do we really want backtics around the quotes here? That seems like unnecessary double-quoting.

Reply:
I just at first had tried to angle for reducing conflicts but that ship has sailed